content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst

```diff
@@ -1,262 +1,349 @@
-
-Customer Lead Time
-------------------
+========================
+Scheduled delivery dates
+========================
+
+Accurately forecasting delivery dates is vital for fulfilling customer expectations. In Odoo, the
+*Inventory* app allows for comprehensive lead time configuration, allowing coordination and planning
+of manufacturing orders, deliveries, and receptions.
+
+Lead time types
+===============
+
+Different lead times for different operations can impact various stages of the order fulfillment
+process. Here's a summary of the types of lead times in Odoo:
+
+.. image:: scheduled_dates/all-lead-times.png
+   :align: center
+   :alt: Show graphic of all lead times working together.
+
+-  **Customer lead time**: The default time frame for fulfilling customer orders. The customer lead
+   time is the number of days from the date the sales order (SO) is confirmed to the date the
+   products are shipped from the warehouse. This is also known as *delivery lead time*.
+   :ref:`[Skip to section] <inventory/management/customer-LT>`
+
+-  **Sales security lead time**: moves the *scheduled delivery date* forward by a specified number
+   of days. This serves as a buffer to allow the team ample time to prepare the outgoing shipment
+   earlier, considering the possibility of delays in the fulfillment process. :ref:`[Skip to
+   section] <inventory/management/sales-security-LT>`
+
+-  **Purchase lead time**: the number of days from the confirmation of a purchase order (PO) to the
+   receipt of products. It provides insight on the time it takes for products to arrive at the
+   warehouse, facilitating effective scheduling and planning of supplier deliveries :ref:`[Skip to
+   section] <inventory/management/purchase-LT>`
+
+-  **Purchase security lead time**: advances the order deadline on a :abbr:`PO (Purchase Order)` by
+   a specified number of days. This proactive approach of placing orders earlier mitigates the risk
+   of vendor or shipping delays. Thus, for products that are set to replenish to order, the need
+   appears on the *Replenishment report* earlier, according to the specified number of days.
+   :ref:`[Skip to section] <inventory/management/purchase-security-LT>`
+
+-  **Manufacturing lead time**: the number of days needed to complete a manufacturing order (MO)
+   from the date of confirmation. This lead time includes weekends (non-working hours in Odoo), and
+   is used to forecast an approximate production date for a finished good. :ref:`[Skip to section]
+   <inventory/management/manuf-LT>`
+
+-  **Manufacturing security lead time**: moves the scheduled date of the :abbr:`MO (Manufacturing
+   Order)` forward by a specified number of days. When used in conjunction with :ref:`replenish to
+   order <inventory/management/products/strategies>`, the security lead time makes the need appear
+   earlier on the replenishment report. :ref:`[Skip to section]
+   <inventory/management/manuf-security-LT>`
```

Comment:
Lets add the internal references inline with the section titles to avoid ambiguous link names with extra formatting like "[Skip to Section]" 🙂

Also, these bullet points have an extra space between the point and the text.

Finally, I will suggest keeping the custom anchor text lowercase.
```suggestion
- :ref:`Customer lead time <inventory/management/customer-lt>`: The default time frame for
  fulfilling customer orders. The customer lead time is the number of days from the date the sales
  order (SO) is confirmed to the date the products are shipped from the warehouse. This is also
  known as *delivery lead time*.

- :ref:`Sales security lead time <inventory/management/sales-security-lt>`: moves the *scheduled
  delivery date* forward by a specified number of days. This serves as a buffer to allow the team
  ample time to prepare the outgoing shipment earlier, considering the possibility of delays in the
  fulfillment process.

- :ref:`Purchase lead time <inventory/management/purchase-lt>`: the number of days from the
  confirmation of a purchase order (PO) to the receipt of products. It provides insight on the time
  it takes for products to arrive at the warehouse, facilitating effective scheduling and planning
  of supplier deliveries

- :ref:`Purchase security lead time <inventory/management/purchase-security-lt>`: advances the order
  deadline on a :abbr:`PO (Purchase Order)` by a specified number of days. This proactive approach
  of placing orders earlier mitigates the risk of vendor or shipping delays. Thus, for products that
  are set to replenish to order, the need appears on the *Replenishment report* earlier, according
  to the specified number of days.

- :ref:`Manufacturing lead time <inventory/management/manuf-lt>`: the number of days needed to
  complete a manufacturing order (MO) from the date of confirmation. This lead time includes
  weekends (non-working hours in Odoo), and is used to forecast an approximate production date for a
  finished good.

- :ref:`Manufacturing security lead time <inventory/management/manuf-security-lt>`: moves the
  scheduled date of the :abbr:`MO (Manufacturing Order)` forward by a specified number of days. When
  used in conjunction with :ref:`replenish to order <inventory/management/products/strategies>`, the
  security lead time makes the need appear earlier on the replenishment report.
```